content/applications/general/companies.rst

```diff
@@ -95,41 +202,50 @@ First, make sure each one of your companies is properly set in relation to:
 - :doc:`Fiscal Localizations <../finance/fiscal_localizations>`
 - :doc:`Pricelists <../sales/sales/products_prices/prices/pricing>`
 
-Now, activate the *Inter-Company Transactions* option under *Settings*. With the respective company
-*activated* and *selected*, choose if you would like operations between companies to be synchronized
-at an invoice/bills level or at a sales/purchase orders level.
+Next, navigate to the :guilabel:`Inter-Company Transactions` tab, on the individual company's
+detail form. To do so, go to the :menuselection:`Settings app --> Companies section --> Manage
+Companies`, and select a company from the list.
```

Comment:
Suggestion to move the navigation instructions before the tab configuration, as this is a more logical flow when reading through.
```suggestion
Next, navigate to :menuselection:`Settings app --> Companies section --> Manage Companies`. Then,
select the desired company from the list. On the company form, select the :guilabel:`Inter-Company
Transactions` tab, on the individual company's detail form.
```